modules/ibi-create-config-iso.adoc

```diff
@@ -213,7 +213,7 @@ spec:
 <3> Specify the name of the `ClusterDeployment` resource that you want to use for the image-based installation of the target host.
 <4> Specify the hostname for the cluster.
 <5> Specify the name of the `ClusterImageSet` resource you used to define the container release images to use for deployment.
-<6> Specify the public Classless Inter-Domain Routing (CIDR) of the external network. For dual-stack networking, you can specify both IPv4 and IPv6 CIDRs using a list format. Regardless of the list order, the IPv4 family becomes the primary address family.
+<6> Specify the public Classless Inter-Domain Routing (CIDR) of the external network. For dual-stack networking, you can specify both IPv4 and IPv6 CIDRs using a list format.
```

Comment:
Maybe we should add that the first address is the primary and it must match the primary IP stack of the seed